addons/hr_holidays/report/hr_leave_report.py

```diff
@@ -11,8 +11,9 @@ class LeaveReport(models.Model):
     _auto = False
     _order = "date_from DESC, employee_id"
 
+    leave_id = fields.Integer(readonly=True)
```

Comment:
@thle-odoo I don't know :thinking: I don't know if it's a good practice to have integer field to represent 2 Many2one fields..
Why won't we just add back allocation_id instead, as it was before? What do you think?

Reply:
It was just a suggestion, I have reused the `allocation_id` field which is fine in this case